### PR TITLE
feat: adds live stream component

### LINKED ISSUE #67: Daily livestream
@yuttadhammo How does it work in the current version?

**Update**: Check for http://sirimangalo.org:8000/live

### PR DESCRIPTION
**Pull request by binarious**
Closes #67

### CODE REVIEW

File: src/app/live/live.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<md-card>
+  <md-card-title>Live Stream</md-card-title>
+
+  <p *ngIf="!liveStream?.online">
+    Live stream currently offline. <br>
+    Next broadcast is <b>Daily Dhamma</b> at <b>0100h UTC</b> (... from now).
```
Comment (olastor) [inline src/app/live/live.html]: What does '(... from now)' mean?

Comment (binarious) [inline src/app/live/live.html]: Whoops, the conversion to local time is missing.

Comment (binarious) [inline src/app/live/live.html]: @olastor Fixed.

File: src/app/home/home.html [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+    [routerLink]="['/home', { tab: 'chat' }]"
+  >
     CHAT
   </button>
-  <button md-button [class.highlight]="currentTab === 'commit'" (click)="tab('commit')">
+  <button
+    md-button
+    [class.highlight]="currentTab === 'commit'"
+    [routerLink]="['/home', { tab: 'commit' }]"
+  >
```
Comment (olastor) [inline src/app/home/home.html]: Aren't those the changes from #145 (already in master)? Could you please rebase on current master?

Comment (binarious) [inline src/app/home/home.html]: @olastor Fixed.

**Pull request merged**